code/__DEFINES/subsystems.dm

```diff
@@ -66,15 +66,15 @@
 #define INIT_ORDER_CIRCUIT 4
 #define INIT_ORDER_TIMER 1
 #define INIT_ORDER_DEFAULT 0
-#define INIT_ORDER_AIR -1
+#define INIT_ORDER_SHUTTLE -1
+#define INIT_ORDER_AIR -2
 #define INIT_ORDER_MINIMAP -3
 #define INIT_ORDER_ASSETS -4
 #define INIT_ORDER_ICON_SMOOTHING -5
 #define INIT_ORDER_OVERLAY -6
 #define INIT_ORDER_XKEYSCORE -10
 #define INIT_ORDER_STICKY_BAN -10
 #define INIT_ORDER_LIGHTING -20
-#define INIT_ORDER_SHUTTLE -21
```

Comment:
Why didnt you just put air after here?

Reply:
i am a very intelligent man but anyway does it really matter?

Reply:
Well then it wouldnt have a lighting issue if it did. Apart from that no idea.

Reply:
meh